docs/software/integrations/mqtt/index.mdx

```diff
@@ -1,8 +1,9 @@
 ---
 id: mqtt
-title: MQTT
+title: MQTT | MQTT Integrations Overview
```

Comment:
Do you feel it's necessary to repeat `MQTT` twice here? Wondering if we can shorten to `MQTT | Integration Overview`